composer.json

```diff
@@ -22,7 +22,7 @@
         "drupal/migrate_report": "~1.0-alpha1",
         "drupal/migrate_run": "~1.0-alpha1",
         "drupal/migrate_spreadsheet": "~1.0-beta1",
-        "drupal/og": "dev-1.x#34a9425ffeeb267ceeba21261f4719d0ced36e58",
+        "drupal/og": "dev-ISAICP-2709 as dev-1.x",
```

Comment:
I checked the PR in Og. 
Are you sure this ensures us?
You are adding the 'administer group' permission to the facilitator. This means that no matter what happens, when a new menu item or a new addition to the admin UI is added upstream, the facilitator will automatically acquire new abilities which we do not handle or currently have a way to see immediately.

I suggest that we either duplicate the form or at least for now, add a small test to ensure that only one menu item called 'Members' is found in the admin form. At least that ensures us that we get changes right away (though the first option is better).